packages/nimble-components/src/table-column/enum-base/models/mapping-icon-config.ts

```diff
@@ -9,6 +9,7 @@ export interface IconView {
 const createIconTemplate = (icon: string): ViewTemplate<IconView> => html`
     <${icon}
         title="${x => x.text}"
+        role="graphics-symbol"
```

Comment:
Can you also add a role to the grouped icon in the table header? https://github.com/ni/nimble/blob/ab995f09749e33f9fdcb5a5975a8c1d84d8aca00/packages/nimble-components/src/table/components/header/template.ts#L40

Reply:
Any idea why we don't also make the ascending/descending sort indicators accessible?